### ISSUE TITLE
Mixed absolute and relative paths in the output

### ISSUE THREAD

**Issue by vilic** [palantir/tslint#2462]
### Bug Report

- __TSLint version__: 5.0.0
- __TypeScript version__: 2.2.2
- __Running TSLint via__: (pick one) CLI / VSCode

Command:

```sh
tslint -c tslint.json -p tsconfig.json --type-check
```

Part of the output:

```
ERROR: C:/Projects/clime/src/castable/array.ts[46, 9]: Operands of '+' operation must either be both strings or both numbers
ERROR: C:/Projects/clime/src/castable/fs.ts[25, 14]: require() style import is forbidden
ERROR: C:/Projects/clime/src/validation/number.ts[8, 9]: Operands of '+' operation must either be both strings or both numbers
ERROR: src/examples/multi-level/cli.ts[9, 1]: Promises must be handled appropriately
ERROR: src/examples/multi-root/cli.ts[15, 1]: Promises must be handled appropriately
ERROR: src/examples/single-level/cli.ts[7, 1]: Promises must be handled appropriately
```

Possibly related to #1794.

**Comment by vilic** [comment]
Still an issue on 5.1.0, can you guys take a look? @andy-hanson

**Comment by nchen63** [comment]
can you provide a repro? I don't see this on my machine

**Comment by vilic** [comment]
@nchen63 Please try:

```sh
git clone https://github.com/vilic/vio.git
npm install
git checkout 96d3cbbf3376b177a12b584877fa30365cd38555
npm run lint
```
